substrate/frame/balances/src/lib.rs

```diff
@@ -1248,5 +1266,42 @@ pub mod pallet {
 			});
 			Ok(actual)
 		}
+
+		/// Generate dev account from derivation string.
+		pub fn derive_dev_account(num_accounts: u32, balance: T::Balance, derivation: &str) {
+			// Ensure that the number of accounts is not zero
+			assert!(num_accounts > 0, "num_accounts must be greater than zero");
+
+			assert!(
+				balance >= <T as Config<I>>::ExistentialDeposit::get(),
+				"the balance of any account should always be at least the existential deposit.",
+			);
+
+			assert!(
+				derivation.contains("{}"),
+				"Invalid derivation, expected `{{}}` as part of the derivation"
+			);
+
+			for index in 0..num_accounts {
+				// Replace "{}" in the derivation string with the index.
+				let derivation_string = derivation.replace("{}", &index.to_string());
+
+				// Attempt to create the key pair from the derivation string with error handling.
+				let pair: SrPair = Pair::from_string(&derivation_string, None)
```

Comment:
Please highlight in the docs that it uses `sr25519` since there are different cryptos available and they make a difference in the generated account.